bats/tests/extensions/install.bats

```diff
@@ -38,6 +38,7 @@ encoded_id() { # variant
 @test 'start container engine' {
     RD_ENV_EXTENSIONS=1 start_container_engine
     wait_for_container_engine
```

Comment:
We should no longer need this call

Reply:
No, because having this call means we ensure forward progress without having everything be stuck on one timeout.  This is important as things can take time to happen; otherwise we'd need a longer timeout when waiting for the extension manager. (Remember that the extension manager launches _after_ everything else, including Kubernetes.)

Reply:
In that case I would (maybe) add the `wait_for_container_engine` call inside the `wait_for_extension_manager` function, so we always call it, and not just sometimes.